src/test/scala/org/scalameter/MethodInvocationCountTest.scala

```diff
@@ -0,0 +1,7 @@
+package org.scalameter
+
+class MethodInvocationCountTest extends InvocationCountMeasurerTest {
+  test("MethodInvocationCountTest.allocations should be deterministic") {
+    checkInvocationCountMeasurerTest(new org.scalameter.examples.MethodInvocationCountTest1)
```

Comment:
Btw, I've just noticed that `Initialization` and `JavaPerformanceTest` share the same `executeTests` body - could it be a good idea to just move it to one base class and declare `executeTests` as final?